lib/ecto/adapters/postgres/connection.ex

```diff
@@ -183,6 +183,9 @@ if Code.ensure_loaded?(Postgrex) do
     defp on_conflict({:nothing, _, targets}) do
       "ON CONFLICT " <> conflict_target(targets) <> "DO NOTHING"
     end
+    defp on_conflict({:replace_all, _, targets}) do
+      error!(nil, "The :replace_all option is not supported by PostgreSQL")
```

Comment:
@josevalim Thank you kindly!
I was not using PostgresSQL in my Project, I did not support it, but also understand the syntax of PostgresSQL, Thank you.
I will try the integration test, but I will contact you if I'm in trouble ;)